auth-lib/build.gradle

```diff
@@ -20,12 +20,10 @@
  */
 
 apply plugin: 'com.android.library'
-apply plugin: 'com.github.dcendents.android-maven'
 
-/* Maven properties */
-group = 'com.spotify.sdk'
-version = '1.0.0'
-project.archivesBaseName = 'spotify-android-auth'
+project.group = 'com.spotify.android'
+project.archivesBaseName = 'auth'
+project.version = '1.0.0-alpha'
```

Comment:
@d-kunin @erdtman should this be 1.0.1, since we already kind of have 1.0.0?
https://github.com/spotify/android-auth/releases/tag/1.0